### DISCUSSION TITLE
onFieldInputValueChange 和 onFieldValueChange的区别

### DISCUSSION THREAD

**Discussion by MatriXiao88** [alibaba/formily#2523] [Q&A 2.x] [answered]
疑问如下：
1. 同样都是在监听字段值变化的场景，通过A控件值的变化，需要经过逻辑代码的处理后去设置B控件值，那么应该用哪个？
2. 源码里面是怎么去区分InputValue和Value的，或者这两者有啥区别，在我看来，例如对于Input组件，两者应该都是一样的吧？

**Comment by janryWang** [comment, marked as answer]
ValueChange是如果联动修改字段值会触发，InputValueChange是只有用户输入的时候才会触发
